README.md

```diff
@@ -36,7 +36,7 @@ The *ArcGIS Runtime Toolkit for iOS* has a *Target SDK* version of *13.0*, meani
  1. Open your project in Xcode
  2. Go to *File* > *Swift Packages* > *Add Package Dependency* option 
  3. Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL
- 4. Choose version 100.11.0 or a later version. Click Next. Only version 100.11.0 or newer supports Swift Package Manager.
```

Comment:
Should we keep that last sentence since when you add the Toolkit SP to a project you can still see the other versions that don't support SPM.

Reply:
Done.